docs/reference/sql/limitations.asciidoc

```diff
@@ -197,4 +197,9 @@ The values that the <<sql-syntax-pivot, `PIVOT`>> query could pivot must be prov
 SELECT * FROM test_emp PIVOT (SUM(salary) FOR languages IN (1, 2))
 -------------------------------------------------------------
 
-the `languages` of interest must be listed explicitly: `IN (1, 2)`.
+the `languages` of interest must be listed explicitly: `IN (1, 2)`. On the other hand, this example would **not work**:
+
+[source, sql]
+-------------------------------------------------------------
+SELECT * FROM test_emp PIVOT (SUM(salary) FOR languages IN (SELECT languages FROM test_emp WHERE languages <=2 GROUP BY languages))
```

Comment:
I think it would be better to have something like: ```... IN (SELECT languages FROM test_emp WHERE salary > 10000)```

Reply:
I wanted to have a subquery that would produce the the same result set as the working query.
But what's the thought behind your example?

Reply:
I get your point, my thinking is that this feels like artificial and not "real life" query.

Reply:
I see. I would hope that the "working" example meets the "realistic" requirements.
The proposed alternative for the "not working" is less convoluted, true, but IMO it would required support for "unique" / "distinct" to bring it closer to the results of the "working" example and the implicit grouping that actually PIVOT does, before applying the IN-filtering.